client/src/components/content/PropertiesPanel/PropertiesPanel.jsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState, useEffect } from 'react';
+import PropertiesPanelView from './PropertiesPanelView/PropertiesPanelView';
+import './PropertiesPanel.css';
+import fetchTaskParametersAndUpdateRPAProperties from '../../../utils/xmlUtils';
+import {
+  fetchTasksFromDB,
+  getAvailableApplications,
+} from '../../../api/rpaFramework';
+import { Typography } from 'antd';
+
+const { Title } = Typography;
+
+/**
+ * @component
+ * @description This class decides which sidebar is displayed. It updates itself depending on the number of selected BPMN elements.
+ * @description Initializes state based on properties; initializes session storage. Binds all state-methods.
+ */
```

Comment:
I'm not sure if it renders that properly. Please use the `@description` tag and just continue writing below it (as in line 105-107)

Reply:
You are right. I will do that.